docs/esa/hsa/hsa.rst

```diff
@@ -20,15 +20,17 @@ Examples
 1. Getting Herschel data
 ------------------------
 
-.. doctest-remote-data::
+.. Skipping becuase of how long the download takes
```

Comment:
I wonder whether it's possible to identify an observation_id that is much smaller, for here and below? cc @jespinosaar